SearchBayesianSkopt: Config 0 Exception. Config: {'topK': 626, 'shrink': 404, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.7543772416615548, 'tversky_beta': 1.0580494663766782} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 1 Exception. Config: {'topK': 199, 'shrink': 523, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.6695085354548307, 'tversky_beta': 1.301445017242079} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 2 Exception. Config: {'topK': 141, 'shrink': 602, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.5485982182740854, 'tversky_beta': 1.346466055574799} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 3 Exception. Config: {'topK': 561, 'shrink': 323, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.549739701913054, 'tversky_beta': 1.7899193196875627} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 4 Exception. Config: {'topK': 549, 'shrink': 982, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.6621684929432692, 'tversky_beta': 1.7772710102281388} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 5 Exception. Config: {'topK': 989, 'shrink': 69, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.10502688332885418, 'tversky_beta': 0.04157929551907953} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 6 Exception. Config: {'topK': 29, 'shrink': 927, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.09601983091442604, 'tversky_beta': 0.05509731627220728} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 7 Exception. Config: {'topK': 945, 'shrink': 37, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.973859028884007, 'tversky_beta': 0.10739780824036929} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 8 Exception. Config: {'topK': 981, 'shrink': 991, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.13873280362545015, 'tversky_beta': 1.933785619926434} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 9 Exception. Config: {'topK': 29, 'shrink': 46, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.9882188730124646, 'tversky_beta': 1.900959592495999} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 10 Exception. Config: {'topK': 13, 'shrink': 915, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.059184684794263434, 'tversky_beta': 0.01944100408299532} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 11 Exception. Config: {'topK': 993, 'shrink': 974, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.6257856338174836, 'tversky_beta': 0.0627678146656765} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 12 Exception. Config: {'topK': 881, 'shrink': 0, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.06517779723458574, 'tversky_beta': 1.8596981336992586} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 13 Exception. Config: {'topK': 22, 'shrink': 66, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.014311521547091036, 'tversky_beta': 0.004884494069526958} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 14 Exception. Config: {'topK': 962, 'shrink': 945, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.9392699754330975, 'tversky_beta': 0.0075192679498532495} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 15 Exception. Config: {'topK': 15, 'shrink': 989, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.857162229854554, 'tversky_beta': 1.9001107996674773} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 16 Exception. Config: {'topK': 98, 'shrink': 31, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.009580103683169004, 'tversky_beta': 1.961784303819482} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 17 Exception. Config: {'topK': 31, 'shrink': 33, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.8376668488538075, 'tversky_beta': 0.043640786763898605} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 18 Exception. Config: {'topK': 993, 'shrink': 59, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.011837844089349184, 'tversky_beta': 1.98341379101803} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 19 Exception. Config: {'topK': 979, 'shrink': 82, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.994631961440198, 'tversky_beta': 1.949233702651787} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 20 Exception. Config: {'topK': 948, 'shrink': 997, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.1647099988538652, 'tversky_beta': 1.9255916918504352} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 21 Exception. Config: {'topK': 908, 'shrink': 17, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.9936978083524577, 'tversky_beta': 0.009239265741175392} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 22 Exception. Config: {'topK': 36, 'shrink': 953, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.13627608374228578, 'tversky_beta': 0.06566065574978777} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 23 Exception. Config: {'topK': 39, 'shrink': 42, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.9490217493189055, 'tversky_beta': 1.9682706046918739} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 24 Exception. Config: {'topK': 980, 'shrink': 954, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.9806506959115342, 'tversky_beta': 0.037640687013410286} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 25 Exception. Config: {'topK': 6, 'shrink': 917, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.9842008378733464, 'tversky_beta': 1.96972637817656} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 26 Exception. Config: {'topK': 93, 'shrink': 7, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.05770724354300308, 'tversky_beta': 0.011822762530762924} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 27 Exception. Config: {'topK': 835, 'shrink': 987, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.009627016189683934, 'tversky_beta': 0.026380290701465546} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 28 Exception. Config: {'topK': 19, 'shrink': 986, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.0006624312493861418, 'tversky_beta': 0.0548263672069076} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 29 Exception. Config: {'topK': 903, 'shrink': 34, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.06434994309247442, 'tversky_beta': 1.9591792950394433} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 30 Exception. Config: {'topK': 13, 'shrink': 3, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.9674705638856484, 'tversky_beta': 0.0354410575662183} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 31 Exception. Config: {'topK': 963, 'shrink': 996, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.8201234786687621, 'tversky_beta': 1.998449087137011} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 32 Exception. Config: {'topK': 988, 'shrink': 39, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.004900476059929055, 'tversky_beta': 0.1944940988776924} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 33 Exception. Config: {'topK': 21, 'shrink': 908, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 0.006340364015669636, 'tversky_beta': 1.9425549525014685} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Config 34 Exception. Config: {'topK': 973, 'shrink': 991, 'similarity': 'tversky', 'normalize': True, 'tversky_alpha': 1.8459773279973757, 'tversky_beta': 0.040639853734443086} - Exception: Traceback (most recent call last):
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 335, in _objective_function
    current_fit_parameters_dict)
  File "C:\Users\ALEXDRUSO\Desktop\RecSysChallenge2020\src\ParameterTuning\SearchAbstractClass.py", line 242, in _evaluate_on_validation
    result_dict, _ = self.evaluator_validation.evaluateRecommender(recommender_instance)
TypeError: evaluateRecommender() missing 1 required positional argument: 'recommender_object'

SearchBayesianSkopt: Search complete. Best config is None: None
